backend-project/small_eod/cases/templates/cases/email/case_created.html

```diff
@@ -0,0 +1,14 @@
+{% extends "notifications/email/mail_base.html" %}
+{% block subject %}
+<p>Utworzono nową sprawę - {{instance.name}}</p>
+{% endblock %}
+
+{% block content %}
+<p>W systemie small_eod pojawiła się nowa sprawa.</p>
+
+<p>
```

Comment:
To jest wystarczające na ten moment. Zaznaczam, że pewnie w przyszłości informacje o sprawie prawdopodobnie będziemy chcieli mieć w jakiejś formie stopki, aby było jedno miejsce do klikania. Pozwólby jednak temu działać i wykorzystać przez Agnieszkę.